lib/src/big_decimal.dart

```diff
@@ -30,6 +30,8 @@ class BigDecimal implements Comparable<BigDecimal> {
     );
   }
 
+  static BigDecimal zero = BigDecimal.fromBigInt(BigInt.zero);
```

Comment:
`BigInt` also offers `.one` and `.two` as static values
Could you add them as well?